packages/react/kibana_context/root/eui_provider.tsx

```diff
@@ -21,11 +22,16 @@ import {
 } from '@kbn/react-kibana-context-common';
 import { ThemeServiceStart } from '@kbn/react-kibana-context-common';
 
+interface IUserProfile {
+  getUserProfile$: () => Rx.Observable<Record<string, unknown> | null>;
```

Comment:
curious why this is custom instead of picking needed props from UserProfileService `UserProfileService['getUserProfile$']`

Reply:
Cleaned up in 1b4e99e69d7